global/utms@3.0-community-project.js

```diff
@@ -0,0 +1,57 @@
+// UTM tracking
```

Comment:
utms@3.0 wraps this all in `(function() { ... })()` which is better. Removes it from global namespace.